### ISSUE TITLE
hard parameter limits prevent from using sensible values

### ISSUE THREAD

**Issue by wohltat** [fieldOfView/Cura-ArcWelderPlugin#12]
if the chosen values for the cura arcwelder plugin parameters exceed the minimum or maximum values, then the plugin refuses to process completely. This is no good behaviour. Especially because it prevents from using sensible values for parameters like
```
    "arcwelder_resolution":
    {
        "minimum_value": 0.025,
        "maximum_value": 0.075,
```
There are definitely usable values outside that range. 

A better way would be make soft limits that give a warning but to not stop the processing.
Alternatively just drop the limits altogether and put the hints in the parameter description.

**Comment by fieldOfView** [comment]
You are quite right. I think the minimum and maximum were introduced as a misinterpretation of the default value mentioned here:
https://github.com/FormerLurker/ArcWelderLib#resolution-maximum-path-deviation

'Default: 0.05 (+- 0.025)' was interpreted as the value for the setting being limited. It would seem that is incorrect. I'll remove the artificial limits.

**Comment by fieldOfView** [comment]
Fixed in 826d568, but I'll leave this issue open until a version containing this fix is released in the Marketplace.

**Comment by wohltat** [comment]
is it not released in the marketplace yet?
